client/js/components/auth/login.js

```diff
@@ -0,0 +1,21 @@
+const html = require('choo/html')
+const form = require('get-form-data')
+
+const login = (state, prev, send) => {
+  function onSubmit (e) {
```

Comment:
yeah JS is a mess right now, that's why it's even more important to work on code style